.github/workflows/update-index.yaml

```diff
@@ -18,6 +18,11 @@ on:
   workflow_dispatch:
 
 jobs:
+
+  enable_pack_release_workflow:
+    name: 'Enable Pack Release Workflow'
+    uses: StackStorm-Exchange/ci/.github/workflows/pack-enable_release_workflow.yaml@master
```

Comment:
Doing this every 5 minutes with the index update workflow is a bit excessive.
Could we run this job in a separate workflow once a month?

Reply:
> Doing this every 5 minutes with the index update workflow is a bit excessive. Could we run this job in a separate workflow once a month?

Hehe, totally.